### PR TITLE
update pizdaint data

### PR DESCRIPTION
**Pull request by psychocoderHPC**
- update profile exmaple
- update tbg templates
- rename `total.tpl` to `large.tpl` (the queue names has changed with the last pizdaint upgrade)

## Tests

- [x] pizdaint 05/2017
- [x] X30 -> X50: only 12 CPU cores / node (.tpl files, too)

### CODE REVIEW

File: src/picongpu/submit/pizdaint-cscs/picongpu.profile.example [hunk 1]
```diff
@@ -46,7 +46,7 @@ export CRAY_CPU_TARGET=x86-64
 export CC=`which cc`
 export CXX=`which CC`
 
-export PICSRC=$HOME/src/picongpu
+export PICSRC=$HOME/picongpu
```
Comment (ax3l) [inline src/picongpu/submit/pizdaint-cscs/picongpu.profile.example]: please keep this in `$HOME/src` as in our examples :)

**Comment by ax3l** [comment]
@psychocoderHPC yes, only 12 cores per node: http://www.cscs.ch/computers/piz_daint/index.html

**Comment by ax3l** [comment]
@psychocoderHPC with 16 GB global, do we need to evaluate if we might better take smaller particle heaps or e.g. 4 MPI host ranks (with individual domains) per P100 or is the 2/4GB problem with slow access to global no more?

**Comment by psychocoderHPC** [comment]
@ax3l Currently we can't support GPU sharing over multiple MPI ranks. 

The issue with the slow random access to global memory is solved with pascal. IMO Pascal support 16 GB RAM random memory access without performance impact.
We have random access memory access pattern in PIConGPU but we are cluster our species data in frame therefore the pressure to the TLB is not critical.

**Pull request merged**